src/test/resources/sql/before-test.sql

```diff
@@ -17,17 +17,19 @@ INSERT INTO defendant
 VALUES
     (-1, -1,'PNC12345', 'CRN1234', 'Iestyn', 'Carey', null, 'Mahoney', 'Male', '1965-06-18', 'test', CURRENT_TIMESTAMP, 'test', CURRENT_TIMESTAMP, 0),
     (-2, -1,'PNC12345', 'CRN1234', 'Iestyn', 'Carey', 'Bob', 'Mahoney', 'Male', '1965-06-18', 'test', CURRENT_TIMESTAMP, 'test', CURRENT_TIMESTAMP, 0),
-    (-3, -2,'PNC12345', 'CRN1234', 'John', null, null, 'Mahoney', 'Male', '1965-06-18', 'test', CURRENT_TIMESTAMP, 'test', CURRENT_TIMESTAMP, 0),
-    (-4, -2,'PNC12345', 'CRN1234', 'John', null, null, 'Mahoney', 'Male', '1965-06-18', 'test', CURRENT_TIMESTAMP, 'test', CURRENT_TIMESTAMP, 0),
+    (-3, -2,'PNC123456', 'CRN1234', 'John', null, null, 'Mahoney', 'Male', '1965-06-18', 'test', CURRENT_TIMESTAMP, 'test', CURRENT_TIMESTAMP, 0),
+    (-4, -2,'PNC123456', 'CRN1234', 'John', null, null, 'Mahoney', 'Male', '1965-06-18', 'test', CURRENT_TIMESTAMP, 'test', CURRENT_TIMESTAMP, 0),
```

Comment:
we should probably use PNCs in our setup data that look like real PNCs (given the amount of validation and general attention they are getting at the moment) unless we are trying to cause validation errors of course.

Reply:
I will add another small ticket to update this later as this affects all the IT  we have.

Reply:
Created this ticket to address this.
https://dsdmoj.atlassian.net/browse/CPR-149